components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -875,8 +890,6 @@ private void sendToFrameworkForLogout(HttpServletRequest request, HttpServletRes
         addSessionDataToCache(sessionDataKey, sessionDTO);
 
 
-        String commonAuthURL = IdentityUtil.getServerURL(FrameworkConstants.COMMONAUTH, false, true);
-
         String selfPath = request.getContextPath();
```

Comment:
Shouldn't this path be tenant qualified in the tenant qualified URL mode?

Reply:
It's not used